src/grid/rtransform.py

```diff
@@ -58,12 +63,12 @@ def generate_radial(self, oned_grid):
         Parameters
         ----------
         oned_grid : OneDGrid
-            one dimensional grid generated for integration purpose
+            An instance of 1D grid.
 
```

Comment:
The name `generate_radial` of this method is more restrictive than it has to be. The outcome may also be a OneDGrid for other things than radial integration.

Reply:
We'll keep this for later in view of merging this PR in short term.